guests/templates/checklist/band_merch_deadline.html

```diff
@@ -230,6 +230,11 @@ <h4 class="modal-title" id="inventory_item_title_{{ item.id or type }}">{{ label
           setVisible('#coverage', isDedicated);
           setVisible('#rockisland', isRockIsland);
 
+          $('#rockisland input, #rockisland textarea, #rockisland select').prop('disabled', !isRockIsland);
```

Comment:
The hidden required form fields were preventing a band from selecting any option other than "Rock Island".